components/scream/src/share/tests/atm_process_tests.yaml

```diff
@@ -4,17 +4,17 @@ Number of Entries: 2
 Schedule Type: Sequential
 
 Process 0:
```

Comment:
Maybe not appropriate for this PR, but recall the list/sequence YAML construction here. It could be more convenient than a bunch of keys named `Process N`.

Reply:
I think I have an open issue, where I plan to change the syntax, and turn it into something like
```
  Schedule Type: Sequential:
  Processes: [Homme, SHOC]
  Homme:
     ...
  SHOC:
    ...
```
I also don't really like `Process N`, especially if there are multiple proc groups, and you see `Process 0` in different nested sublists. I'll go check on the issues to make sure this is on the radar for v1.1.